src/server.js

```diff
@@ -15,22 +20,32 @@ server.connection({
   },
 });
 
-server.register([ Inert, ], (err) => {
-  if(err) throw err;
-
-  server.route([
-    {
-      path: '/{param*}',
-      method: 'GET',
-      handler: {
-        directory: {
-          path: path.join(__dirname, '../public'),
-          index: true,
-          defaultExtension: 'html',
-        },
-      },
-    },
-  ]);
+const cookieOptions = {
+  password: 'thi£siVVs&Rgro1ww$MjmakingthisLON7pkassword',
+  cookie: 'grow-cookie',
+  isSecure: process.env.NODE_ENV === 'PRODUCTION',
+  ttl: 24 * 60 * 60 * 1000,
+};
+
+const fbOptions = {
+  provider: 'facebook',
+  password: 'cookie_encryption_password_secure',
+  clientId: process.env.APP_FACEBOOK_ID,
+  clientSecret: process.env.CLIENT_SECRET,
+  isSecure: process.env.NODE_ENV === 'PRODUCTION',
+};
+
+server.register([ Bell, ], (err) => {
+  if (err) { throw new Error (err); }
+
+  server.auth.strategy('facebook', 'bell', fbOptions);
+});
+
+server.register([ Inert, cookieAuth,], (err) => {
+  if (err) { throw new Error (err); }
+
+  server.auth.strategy('session', 'cookie', 'optional', cookieOptions);
```

Comment:
Then you can override this on endpoints you do not want auth on (ie login page, or fb callback)